test/integration/extensionRequests.test.js

```diff
@@ -0,0 +1,631 @@
+        .send({
+          taskId: taskId2,
+          title: "change ETA",
+          assignee: user.id,
+          oldEndsOn: 1234,
+          newEndsOn: 1235,
+          reason: "family event",
+          status: "PENDING",
+        })
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("Extension Request created successfully!");
+          const extensionRequestId1 = res.body.extensionRequest.id;
+          chai
+            .request(app)
+            .get(`/logs/extensionRequest/?body.assignee=${user.id}&meta.taskId=${taskId2}`)
+            .set("cookie", `${cookieName}=${superUserJwt}`)
+            .end((err, res) => {
+              if (err) {
+                return done(err);
+              }
+              expect(res).to.have.status(200);
+              expect(res.body).to.be.a("object");
+              expect(res.body.message).to.equal("Logs returned successfully!");
+              expect(res.body.logs).to.be.a("array");
+              expect(res.body.logs[0].body.extensionRequestId).to.equal(extensionRequestId1);
+              expect(res.body.logs[0].body.assignee).to.equal(user.id);
+              expect(res.body.logs[0].meta.taskId).to.equal(taskId2);
+              return done();
+            });
+          // eslint-disable-next-line
+          return;
+        });
+    });
+  });
+
+  describe("GET /extensionRequest/:id", function () {
+    it("should return success response and extension request with the id that is provided", function (done) {
+      chai
+        .request(app)
+        .get(`/extensionRequests/${extensionRequestId1}`)
+        .set("cookie", `${cookieName}=${superUserJwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.be.equal("Extension Requests returned successfully!");
+          expect(res.body.extensionRequest).to.be.a("object");
+          expect(res.body.extensionRequest.assignee).to.equal(appOwner.username);
+          expect(res.body.extensionRequest.id).to.equal(extensionRequestId1);
+          return done();
+        });
+    });
+    it("should return failure response if no extension request found with :id", function (done) {
+      chai
+        .request(app)
+        .get(`/extensionRequests/1234567890`)
+        .set("cookie", `${cookieName}=${superUserJwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(404);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.be.equal("Extension Request not found");
+          return done();
+        });
+    });
+  });
+
+  describe("GET /extensionRequest", function () {
```

Comment:
since this API is only to be hit by the super user. can we have a test when a normal user tries to use this API.